From 223986aab17f9586f79070b1aeaf959e91f031aa Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 03 Nov 2020 18:33:06 +0200
Subject: [PATCH] [SW-12754] gaudi: security: disable pcie dbi close

The host access to the PCIe DBI must be solved first, otherwise host is
unable to configure iATU via the ELBI and it renders the card useless.
Meanwhile - temporarily - comment out the PCIe DBI and others using the
RR_4 and leave open.

Change-Id: I43c8cc4ceab0dffe337bcd62dda5db240e96276d
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 70eea09..ad851da 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1127,12 +1127,13 @@
 	preboot_secure_lbw_rr(3, min, max);
 	preboot_secure_pcie_rr(3, min, max);
 
+	/* TODO: the below is in comment till we solve the ELBI access ticket */
 	/* PCIE_DBI up to MMU_UP: RR_4 */
-	min = mmPCIE_DBI_BASE - 1;
+/*	min = mmPCIE_DBI_BASE - 1;
 	max = mmMMU_UP_BASE;
 	preboot_secure_lbw_rr(4, min, max);
 	preboot_secure_pcie_rr(4, min, max);
-
+*/
 	/* PSOC_I2C_M0 up to PSOC_TIMESTAMP: RR_5 */
 	min = mmPSOC_I2C_M0_BASE - 1;
 	max = mmPSOC_TIMESTAMP_BASE;
